packages/server/src/models-abstract/template-model.js

```diff
@@ -13,6 +16,23 @@ class TemplateModel extends ModelPostgres {
     });
   }
 
+
+  /**
+   * It is a temporary solution.
```

Comment:
Is there any story for a better solution, by chance?

Reply:
Need to think